Framework/test/testInfrastructureGenerator.cxx

```diff
@@ -85,6 +85,17 @@ TEST_CASE("qc_factory_local_test")
   }
 }
 
+TEST_CASE("onlyTask")
```

Comment:
I think this is tested in the `testSharedConfig.json` as well, "defTask" and "recoTask" do not have associated checks there. I guess we did not discover it was wrong previously because of the wrong update of `qc-check`'s expected number of inputs.
Up to you if you want to remove this one though.